automerge-backend/src/backend.rs

```diff
@@ -96,6 +96,16 @@ impl Backend {
         self.make_patch(diffs, actor)
     }
 
+    fn apply_without_patch(&mut self, changes: Vec<Change>) -> Result<(), AutomergeError> {
```

Comment:
I think it might be worth adding a little docstring here describing why you might do this.

Reply:
Yep, that makes sense. Added!